crates/cheatcodes/src/test.rs

```diff
@@ -38,6 +38,8 @@ impl Cheatcode for zkRegisterContractCall {
             name: name.clone(),
             zk_bytecode_hash: zkBytecodeHash.0.into(),
             zk_deployed_bytecode: zkDeployedBytecode.clone(),
+            //TODO: add argument to cheatcode
+            zk_factory_deps: vec![],
```

Comment:
Can we have a task created for this as well?

Reply:
That exists already on the notion